docs/data/sdks/android-kotlin/index.md

```diff
@@ -18,18 +18,27 @@ The Kotlin Android SDK lets you send events to Amplitude. This library is open-s
 
 ## Getting started
 
+Check out a sample project [here](https://github.com/amplitude/Amplitude-Kotlin/tree/main/samples/kotlin-android-app).
```

Comment:
```suggestion
Check out an [example Android project](https://github.com/amplitude/Amplitude-Kotlin/tree/main/samples/kotlin-android-app).
```
For accessibility, we aim to make hyperlinks clear and distinct so anyone reading _just the link text_ will have an idea of where it's going to take them. I will usually do a mental exercise where I imagine the user is using a screenreader, and ask does the hyperlinked text make sense? If you're curious about the standard, you can read about it here: https://www.boia.org/wcag2/cp/2.4.4 (no pressure, I just like giving 'sources' for my 'rules' 😂 

Also, consider adding a little bit about why they should check out the Android project. Why are we offering that resource to them? Will it help them get started faster? Will it give them ideas of how to instrument? 
Maybe aim for something like this: 

> For instrumentation ideas, check out an [example Android project](https://github.com/amplitude/Amplitude-Kotlin/tree/main/samples/kotlin-android-app).

> To get started fast, check out an [example Android project](https://github.com/amplitude/Amplitude-Kotlin/tree/main/samples/kotlin-android-app).

Reply:
Good to learn the hyperlink standard and totally agree with that. Asking those questions also helps me when writing and have a more clear structure. 

Resolved  as I changed it in quickstart guide